src/languages/es-ES.ts

```diff
@@ -288,10 +288,15 @@ export default class extends Language {
 		COMMAND_PLAYING_QUEUE_NOT_PLAYING: `Creo que estás escuchando ruido de fondo, no estoy reproduciendo nada.`,
 		COMMAND_REPEAT_DESCRIPTION: `Se alterna repitiendo la canción actual.`,
 		COMMAND_REPEAT_SUCCESS: enabled => enabled ? `This is your JAM isn't it? No te preocupes, repetiremos esto una y otra vez!` : `En realidad, también me estaba cansando de esto, pero no quería decir nada.`,
-		COMMAND_QUEUE_DESCRIPTION: `Revisa la lista de cola.`,
-		COMMAND_QUEUE_EMPTY: '¡La cola está vacía! Pero puedes añadir algunas canciones usando el comando `Skyra, add`.',
-		COMMAND_QUEUE_LINE: (title, requester) => `*${title}*, pedida por: **${requester}**`,
-		COMMAND_QUEUE_TRUNCATED: amount => `Mostrando 10 canciones de ${amount}`,
+		COMMAND_QUEUE_DESCRIPTION: `Check the queue list.`,
+		COMMAND_QUEUE_EMPTY: `The session is over, add some songs to the queue, you can for example do \`Skyra, add Imperial March\`, and... *dumbrolls*!`,
+		COMMAND_QUEUE_LAST: `There are no more songs! After the one playing is over, the session will end!`,
+		COMMAND_QUEUE_TITLE: guildname => `Music queue for ${guildname}`,
+		COMMAND_QUEUE_LINE: (position, title, url, duration, requester) => `[${position}] [${title}](${url}) | (${duration}) | Requested by **${requester}**.`,
+		COMMAND_QUEUE_NOWPLAYING: (title, url, duration, requester) => `[${title}](${url}) | (${duration}) | Requested by **${requester}**.`,
+		COMMAND_QUEUE_NOWPLAYING_TITLE: 'Now Playing:',
+		COMMAND_QUEUE_TOTAL_TITLE: 'Total songs:',
+		COMMAND_QUEUE_TOTAL: (songs, remainingTime) => `${songs} song${songs === 1 ? '' : 's'} in the queue, with a total duration of ${remainingTime}`,
```

Comment:
https://translate.google.com

Reply:
Last time I used that, RAE and some kittens died. Skipping...